broker_db/Dockerfile

```diff
@@ -0,0 +1,21 @@
+# Container definition for MassVaxx database to be used for testing 
+# by Broker back-end server.
+#
+# For production, user/passwords should be moveed to secrets file as
```

Comment:
"moved" is misspelled. I tried to fix it myself to avoid bothering you, but everything started blowing up when I did, so I don't know what's going on.

Reply:
If the Broker DB is not running, which I'm guessing is the case, then you will see database connection errors. I'll make sure the files have EOF or whatever is needed for Windows (they work on Linux and macOS but should also on Windows as that is the main system for most of the developers).

Reply:
I think the problem is the opposite. The only time the line ending problem rears its ugly head is when I try to run the broker_db docker container. It tries to run massvaxx_db_init.sh, runs into the CRLF at the end of the first line, and crashes with this error: "/usr/local/bin/docker-entrypoint.sh: /docker-entrypoint-initdb.d/massvaxx_db_init.sh: /bin/bash^M: bad interpreter: No such file or directory". Is there a way to turn off automatic line ending conversion in git for just one file? Or to ensure LF line endings in general?